nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-{
-  "name": "Acadia National Park",
-  "params": [
-    "visibility": {
-      "summariesByYear": [
-        {
-          "year": 2000,
-          "conditionStatus": "Warrants Moderate Concern",
-          "conditionTrend": "Relatively Unchanged",
-          "conditionConfidence": "High Confidence",
-          "conditionSummaryCombined": "concern-unchanging-high",
```

Comment:
This would be used to build up the dynamic icon: 
![screen shot 2018-11-09 at 12 55 10 pm](https://user-images.githubusercontent.com/3201135/48279479-b7498280-e41e-11e8-8651-40eaa9a4b346.png)
If the concern/trend/confidence text is always the same I can generate the icon on my end. Or you could pass a string like this example here where the segments are:
```
"[good/concern/danger]-[improving/unchanging/degrading]-[low/medium/high]"
```
(or something similar. We can figure this out).

Reply:
I'll try the segmented approach.